Access Medicine.js

```diff
@@ -0,0 +1,264 @@
+{
+	"translatorID": "60e55b65-08cb-4a8f-8a61-c36338ec8754",
+	"label": "Access Medicine",
+	"creator": "Jaret M. Karnuta",
+	"target": "^https?://(0-)?accessmedicine\\.mhmedical\\.com",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2016-12-26 19:52:12"
+}
+
+function detectWeb(doc, url){
+	var citation = doc.getElementById("hfGetCitation");
+	if(citation){
+		return "bookSection";
+	}
+}
```

Comment:
There is [`Z.monitorDOMchanges`](https://github.com/zuphilip/translators/wiki/Useful-functions#monitor-dom-changes) for such cases, but I don't if it is worth here to look into this....